test/unit_util.jl

```diff
@@ -85,5 +87,15 @@ for _dummy_ in 1:1 # avoid global variable warnings
         A[:, 1] = A[:, 5] = A[:, n] = zeros(N, m)
         B = LazySets.delete_zero_columns!(A)
         @test size(B) == (m, n-3)
+
+        # extend by orthogonal complement
+        M = N[1 1; 2 2; 3 4.]
+        @assert rank(M) == 2
+        Mext, inv_Mext = extend(M)
+        @test rank(Mext) == 3
+        @test Mext * inv_Mext ≈ I(3)
```

Comment:
`I(3)` needs to be transformed to a `Matrix`.

Reply:
The callable uniform scaling produces a diagonal matrix (with booleans) and it can be compared numerically,

```julia
julia> using LinearAlgebra

julia> a = rand(2, 2); a * inv(a) ≈ I(2)
true
```

Reply:
I just wanted to point out the error [here](https://travis-ci.org/JuliaReach/LazySets.jl/jobs/646047201#L558).

Reply:
Fair enough, that is an erro for an older Julia version :)